sql/monitoring/telemetry_missing_columns_v1/view.sql

```diff
@@ -0,0 +1,32 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod.monitoring.telemetry_missing_columns_v1` AS
+SELECT
+  TIMESTAMP_TRUNC(submission_timestamp, DAY) AS day,
+  'telemetry' AS document_namespace,
+  REGEXP_EXTRACT(_TABLE_SUFFIX, r "^(.*)_v.*") AS document_type,
+  REGEXP_EXTRACT(_TABLE_SUFFIX, r "^.*_v(.*)$") AS document_version,
+  path,
+  COUNT(*) AS path_count
+FROM
+  `moz-fx-data-shared-prod.telemetry_stable.*`,
+  UNNEST(
+    udf_js.json_extract_missing_cols(
+      additional_properties,
+      ARRAY ["histogram_type"],
+      ARRAY [
+        "activeAddons",
+        "userPrefs",
```

Comment:
Do these still showup in the `additional_properties`?

Reply:
It is for documents that aren't the main ping -- we end up having missing section in a lot of pings because the common ping format isn't being applied consistently.

See the example redash query and look for userPrefs in the top results:

["heartbeat_v4","crash_v4","event_v4","third_party_modules_v4","modules_v4","new_profile_v4","shield_study_v4","testpilot_v4","untrusted_modules_v4","system_addon_deployment_diagnostics_v4","update_v4","shield_study_addon_v4"]